### PR TITLE
DOC-3075: Add support for enterprise changefeeds in Serverless

### PR DESCRIPTION
**Pull request by nickvigilante**
Fixes DOC-3075

### CODE REVIEW

File: cockroachcloud/serverless-faqs.md [hunk 1]
```diff
@@ -148,7 +148,7 @@ We don't recommend `userfile` for `EXPORT` operations. You can either add billin
 
 ### Is change data capture available to me?
 
-Yes, {{ site.data.products.serverless-plan }} clusters have access to [Core Changefeeds](../{{site.versions["stable"]}}/changefeed-examples.html#create-a-core-changefeed).
+Yes, {{ site.data.products.serverless-plan }} clusters have access to [Core Changefeeds](../{{site.versions["stable"]}}/changefeed-examples.html#create-a-core-changefeed). Creating {{ site.data.products.enterprise }} changefeeds on {{ site.data.products.serverless }} is supported with an [{{ site.data.products.enterprise }} license](../v22.1/enterprise-licensing.html).
```
Comment (amruss) [inline cockroachcloud/serverless-faqs.md:151]: I thought all serverless clusters had enterprise licenses? cc @emily-horing

**Review by amruss — APPROVED** [review]
One open question around whether you need an enterprise license to use enterprise features in serverless, but besides that LGTM

File: cockroachcloud/serverless-faqs.md (hunk 1 above)
Comment (nickvigilante) [reply cockroachcloud/serverless-faqs.md:151]: I'll wait for Emily to confirm, but the free forever serverless clusters I imagine would not have an enterprise license.

Comment (emily-horing) [reply cockroachcloud/serverless-faqs.md:151]: Yes, all serverless clusters have access to enterprise features

Comment (nickvigilante) [reply cockroachcloud/serverless-faqs.md:151]: Thanks for confirming. I'll fix.

**Review by kathancox — COMMENTED** [review]
LGTM

**Pull request merged**